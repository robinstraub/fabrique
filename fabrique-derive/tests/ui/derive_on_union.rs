use fabrique_derive::Factory;

#[derive(Factory)]
union Anvil {
    heavy: u32,
    light: u16,
}

fn main() {}