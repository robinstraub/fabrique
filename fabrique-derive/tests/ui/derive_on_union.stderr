error: Factory can only be derived from named structs, union given
 --> tests/ui/derive_on_union.rs:4:1
  |
4 | / union Anvil {
5 | |     heavy: u32,
6 | |     light: u16,
7 | | }
  | |_^
